Frameworks/UIKit/UIButton.mm

```diff
@@ -891,22 +946,18 @@ - (UIImageView*)imageView {
 }
 
 /**
- @Status Caveat
- @Notes Returns the receiving view
-*/
-- (UIView*)viewForBaselineLayout {
-    return self;
-}
-
-/**
  @Status Interoperable
 */
 - (CGSize)intrinsicContentSize {
     CGSize ret = CGSizeZero;
 
     UIImage* img = self.currentImage;
     UIEdgeInsets contentInsets = self.contentEdgeInsets;
-    CGSize textSize = [[self currentTitle] sizeWithFont:self.font];
+
+    // TODO: Should we be asking our label for its intrinsicContentSize?
```

Comment:
>// TODO: Should we be asking our label for its intrinsicContentSize? [](start = 4, length = 68)

I guess not. 

intrinsicContentSize

The natural size for the receiving view, considering only properties of the view itself.
[https://developer.apple.com/reference/uikit/uiview?language=objc](https://developer.apple.com/reference/uikit/uiview?language=objc)